app/helpers/permission-to-remove-contributor.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember';
+
+/**
+* permissionToRemoveContributor helper.  Checks to see if user has proper permissions
+* to remove contributor.  The user either must be an admin or trying to remove herself.
+* The project cannot be a registration.
+*/
+export function permissionToRemoveContributor(params/*, hash*/) {
+    var contributor = params[0];
+    var currentUser = params[1];
+    var stillAdmin = params[2];
+    var node = params[3];
+    if (currentUser) {
+        var currentUserId = currentUser.get('currentUserId') || currentUser.get('id');
+        var removeSelf = contributor.id.split('-')[1] === currentUserId;
```

Comment:
Yes, thanks, this was a WIP PR and I hadn't gotten to cleaning this up yet.   This was on my list.  All the same, you've pointed out several things in this PR that I hadn't noticed so I appreciate the thorough review.